src/index.html

```diff
@@ -14,105 +14,142 @@
-          <p class="card__show-answer__answer text-center">Footsteps 🦶🏻</p>
-        </details>
-        <ul class="card__tag-list flex-se">
-          <li class="card__tag-list__item">Tag 1</li>
-          <li class="card__tag-list__item">Tag 2</li>
-          <li class="card__tag-list__item">Tag 3</li>
-          <li class="card__tag-list__item">Tag 4</li>
-        </ul>
-      </section>
+        <section class="card" data-js="card">
+          <i class="card__bookmark fa fa-3x fa-bookmark" data-js="bookmark"></i>
+          <h2>The more you take, the more you leave behind. What am I?</h2>
+          <button class="card__show-answer">Show Answer</button>
+          <p class="card__show-answer__answer text-center hidden">
+            Footsteps 🦶🏻
+          </p>
+          <ul class="card__tag-list flex-se">
+            <li class="card__tag-list__item">Tag 1</li>
+            <li class="card__tag-list__item">Tag 2</li>
+            <li class="card__tag-list__item">Tag 3</li>
+            <li class="card__tag-list__item">Tag 4</li>
+          </ul>
+        </section>
 
-      <section class="card" data-js="card">
-        <i class="card__bookmark fa fa-3x fa-bookmark" data-js="bookmark"></i>
-        <h2>
-          You live in a one story house made entirely of redwood. What color
-          would the stairs be?
-        </h2>
-        <details>
-          <summary class="card__show-answer">Show Answer</summary>
-          <p class="card__show-answer__answer text-center">
+        <section class="card" data-js="card">
+          <i class="card__bookmark fa fa-3x fa-bookmark" data-js="bookmark"></i>
+          <h2>
+            You live in a one story house made entirely of redwood. What colour
+            would the stairs be?
+          </h2>
+          <button class="card__show-answer">Show Answer</button>
+          <p class="card__show-answer__answer text-center hidden">
             What stairs? You live in a one-story house. 🏡
           </p>
-        </details>
+          <ul class="card__tag-list flex-se">
+            <li class="card__tag-list__item">Tag 1</li>
+            <li class="card__tag-list__item">Tag 2</li>
+            <li class="card__tag-list__item">Tag 3</li>
+            <li class="card__tag-list__item">Tag 4</li>
+          </ul>
+        </section>
 
-        <ul class="card__tag-list flex-se">
-          <li class="card__tag-list__item">Tag 1</li>
-          <li class="card__tag-list__item">Tag 2</li>
-          <li class="card__tag-list__item">Tag 3</li>
-          <li class="card__tag-list__item">Tag 4</li>
-        </ul>
-      </section>
+        <section class="card" data-js="card">
+          <i class="card__bookmark fa fa-3x fa-bookmark" data-js="bookmark"></i>
+          <h2>What can you catch but never throw?</h2>
+          <button class="card__show-answer">Show Answer</button>
+          <p class="card__show-answer__answer text-center hidden">A cold 🤧</p>
```

Comment:
Also here: too much nesting and connecting between the elements. Instead of `card__show-answer__answer`, you should only write `card__answer`. There is never a `onething__secondthing__thirdthing`, only `component__element`. For the styling of the "answer" it is irrelevant that there is a button to show it. But it's relevant that the answer is part of the card: if you delete the button, the answer does not care. If you delete the card, there would be no answer anymore as it can't (or is unlikely to) exist without the card.